content/cn/quickstart.html

```diff
@@ -407,7 +407,7 @@ <h2 id="query">查询数据</h2>
 
 <h2 id="updates">更新数据</h2>
 
-<p>这类似于插入新数据。使用数据生成器生成对现有行程的更新，加载到数据帧并将数据帧写入hudi数据集。</p>
+<p>这类似于插入新数据。使用数据生成器生成对现有行程的更新，加载到DataFrame中并将DataFrame写入hudi数据集。</p>
```

Comment:
Please see if there is a DataFrame translation elsewhere, and please revert it to DateFrame.